src/backend/web/routes/auth.js

```diff
@@ -29,40 +28,32 @@ router.post('/login/callback', passport.authenticate('saml'), (req, res) => {
 router.get('/login', passport.authenticate('saml'));
 
 /**
- * Expose a logout method, to provide idP-initiated SLO
- * https://github.com/bergie/passport-saml/issues/221#issuecomment-338896096
+ * /auth/logout/callback is where the external SAML SSO provider will redirect
+ * users upon successful logout.
+ */
+router.get('/logout/callback', (req, res) => {
+  req.logout();
+  res.redirect('/');
+});
+
+/**
+ * /auth/logout allows users to clear login tokens from their session
  */
-function logout(req, res) {
+router.get('/logout', (req, res) => {
   try {
-    // TODO: confirm I can use strategy above... was: const strategy = passport._strategy('saml');
-    strategy.logout(req, (error, requestUrl) => {
+    // eslint-disable-next-line no-underscore-dangle
```

Comment:
interesting way of getting around our ESLint :D. Did not know this was how to do it.

Reply:
Pretend you didn't see this!  I have to do it because Passport doesn't expose the strategy object properly, only as a `._*` param, and our eslint (rightly) says you shouldn't do this, since those are generally private properties not meant to be used externally (only for tests).